src/sqlancer/yugabyte/ysql/YSQLSchema.java

```diff
@@ -160,6 +160,19 @@ protected static List<YSQLColumn> getTableColumns(SQLConnection con, String tabl
         return columns;
     }
 
+    public boolean getDatabaseIsColocated(SQLConnection con) throws Exception {
+        try (Statement s = con.createStatement()) {
+            try (ResultSet rs = s.executeQuery("SELECT yb_is_database_colocated();")) {
+                if (!rs.next()) {
+                    throw new IgnoreMeException();
```

Comment:
Can it actually be that the database system returns an empty result, or would it be a bug in the database system? If it would be a bug, we should not ignore it.

Reply:
I think actually it won't returns an empty result. See: https://www.yugabyte.com/blog/differences-primary-key-usage-colocated-noncolocated-tables/#:~:text=a%20colocated%20database%3A-,SELECT%20yb_is_database_colocated()%3B,-Primary%20key%20syntaxes

Also I test it with and without selecting a databse, and it turned out to return a result.

Reply:
In this case, I think you can remove the check, and thus also remove the exception-handling code in the caller.

Reply:
Ok, make sense.